to.etc.domui/src/main/java/to/etc/domui/dom/Animations.java

```diff
@@ -95,4 +95,33 @@ static public void shake(NodeBase node) {
 			.next();
 	}
 
+	static public void bounce(NodeBase node) {
+		node.appendStatement()
+			.select(node)
+			.append(".effect('bounce')")
+			.next();
+	}
+
+	/**
+	 * Does pulsate effect.
+	 * @param node
```

Comment:
Please do not add param comments if you leave them empty (I'm doing a lot of work to remove those).